x = None
try:
    print(x * 2)
except TypeError as error:
    print(f"エラー内容: {error}")
